routers/user.js

```diff
@@ -0,0 +1,48 @@
+const { Router } = require('express');
+const userRouter = Router();
```

Comment:
could you please move router declaration a few lines down separately from `require` statements?